python/python/glide/glide_client.py

```diff
@@ -516,6 +518,33 @@ class GlideClusterClient(BaseClient, ClusterCommands):
     https://github.com/aws/babushka/wiki/Python-wrapper#redis-cluster
     """
 
+    async def _cluster_scan(
+        self,
+        cursor: ClusterScanCursor,
+        match: Optional[str] = None,
+        count: Optional[int] = None,
+        type: Optional[ObjectType] = None,
+    ) -> List[Union[ClusterScanCursor, List[str]]]:
+        if self._is_closed:
+            raise ClosingError(
+                "Unable to execute requests; the client is closed. Please create a new client."
+            )
+        request = RedisRequest()
+        request.callback_idx = self._get_callback_index()
+        # Take out the id string from the wrapping object
+        cursor_str = cursor.get_cursor()
```

Comment:
If the user accidentally re-submits the final cursor from a previous scan, there should be an error right? (This cursor would have the string "finished").

Reply:
Nope, scan is a stateless command, i tried to stay true to the original command. If the user wants to send the same cursor over and over, that's fine.
It basically will start a new scan from the beginning as in the original scan.